bindata/network/multus/multus.yaml

```diff
@@ -315,7 +315,7 @@ spec:
 {{- if eq .DefaultNetworkType "OpenShiftSDN"}}
         - "--readiness-indicator-file=/var/run/multus/cni/net.d/80-openshift-network.conf"
 {{- else if eq .DefaultNetworkType "OVNKubernetes"}}
-        - "--readiness-indicator-file=/var/run/multus/cni/net.d/10-ovn-kubernetes.conf"
+        - "--readiness-indicator-file=/var/run/multus/cni/net.d/10-ovn-kubernetes.conflist"
```

Comment:
@dougbtv not that I know of?

```
[dcbw@localhost go-controller (master)]$ git grep "10-ovn-k"
cmd/ovn-kube-util/app/readiness-probe.go:       confFile := "/etc/cni/net.d/10-ovn-kubernetes.conf"
pkg/config/config.go:   CNIConfFileName = "10-ovn-kubernetes.conf"
[dcbw@localhost go-controller (master)]$ git grep "conflist"
[dcbw@localhost go-controller (master)]$ 
```

Reply:
Dan -- thank you! Yeah, I think I've got this wrong -- there was a report of this happening, but... I believe it's triggered by the integration of a third party project, so.... Yeah, I need to go back to the drawing board!

Reply:
@dougbtv yeah I've seen that in some other project (Istio?) that hijacks stuff.